src/Sylius/Bundle/UserBundle/DependencyInjection/Configuration.php

```diff
@@ -64,7 +64,24 @@ private function addResourcesSection(ArrayNodeDefinition $node): void
                                             ->arrayNode('token')
                                                 ->addDefaultsIfNotSet()
                                                 ->children()
-                                                    ->scalarNode('ttl')->defaultValue('P1D')->end()
+                                                    ->scalarNode('ttl')
+                                                        ->defaultValue('P1D')
+                                                        ->validate()
+                                                        ->ifTrue(
+                                                            /** @param mixed $ttl */
+                                                            function ($ttl) {
```

Comment:
`mixed` is available since 8.0, should it be used directly?
```suggestion
                                                            function (mixed $ttl) {
```